Makefile.toml

```diff
@@ -614,6 +614,46 @@ docker run --rm \
 '''
 ]
 
+# Provide build options for netdog
+[tasks.netdog-tools]
+dependencies = ["setup", "fetch-sources"]
+script = [
+  '''
+  cargo install \
+    ${CARGO_MAKE_CARGO_ARGS} \
+    --path sources/api/netdog \
+    --root tools \
+    --force \
+    --quiet
+  '''
+]
+
+[tasks.netdog-parse]
+dependencies = ["netdog-tools"]
+script_runner = "bash"
+script = [
+  '''
+  set -euo pipefail
+
+  if [ ! -s "${NET_CONFIG_INPUT}" ]; then
+     echo "No network config file exists, please create one at ${NET_CONFIG_INPUT}"
+     exit 1
+  fi
+
+  docker run --rm \
```

Comment:
It helps to avoid adding in logic just for taking in a file via an argument and instead uses the same lookup logic that `generate-net-config` uses. This all came from when I was going to just call `generate-net-config` and shuffle things in and out of the container to make things *just work* but now that we are creating a separate command for parse-only, maybe it makes sense to just call it with an argument instead.

Reply:
I've removed this from the PR, we will solve this problem a different way.